apps/web/utils/actions/rule.ts

```diff
@@ -46,6 +46,14 @@ import { prefixPath } from "@/utils/path";
 
 const logger = createScopedLogger("actions/rule");
 
+export type CategoryAction =
+  | "label"
+  | "digest"
+  | "label_archive"
+  | "label_digest"
+  | "label_archive_digest"
+  | "none";
```

Comment:
probably just want digest handled separately.
drop down represents 3 values. the digest is a separate toggle.
we could put them all into one dropdown potentially like this, but feels overly messy for the user. i will need to play around with this though a bit